ui/app/styles/components/console-ui-panel.scss

```diff
@@ -18,20 +18,6 @@ $console-close-height: 35px;
   will-change: transform, min-height;
   -webkit-overflow-scrolling: touch;
   z-index: 199;
-
-  .button {
```

Comment:
Leveraging HDS classes to match console button styling

before
![old-console](https://github.com/hashicorp/vault/assets/68122737/8c1fb8a1-41c1-429d-865b-ebdcfe6a677c)

after (now buttons match sidebar nav)
![now-console](https://github.com/hashicorp/vault/assets/68122737/80c1a224-ebf8-42a0-9bd1-cae147ec64b6)